src/main/scala/devices/debug/DebugTransport.scala

```diff
@@ -260,8 +263,14 @@ class DebugTransportModuleJTAG(debugAddrBits: Int, c: JtagDTMConfig)
   tapIO.control.jtag_reset := io.jtag_reset
 
   //--------------------------------------------------------
-  // Reset Generation (this is fed back to us by the instantiating module,
-  // and is used to reset the debug registers).
+  // TAP Test-Logic-Reset state resets the debug registers.
 
-  io.fsmReset := tapIO.output.reset
-}
+  when (tapIO.output.reset) {
```

Comment:
so is this the same as `tapIsInTestLogicReset`...?

Reply:
name changed.